src/index.html

```diff
@@ -1,13 +1,168 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta charset="UTF-8" ></meta>
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    ></meta>
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div
+    class="wrapper"
+    >
+      <div
+      class="stars stars--0"
+      >
+        <div
+        class="stars__star"
+        >
+        </div>
+        <div
+        class="stars__star"
+        >
+        </div>
```

Comment:
formatting is everything, make sure to fix it
```suggestion
    <div class="wrapper">
      <div class="stars stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
```